src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -188,8 +200,9 @@ export class SkyAgGridService {
             ...editableCellClassRules
           },
           cellEditorFramework: SkyAgGridCellEditorDatepickerComponent,
-          valueFormatter: (params: ValueFormatterParams) => this.dateFormatter(params, args.locale),
-          comparator: dateComparator
+          comparator: dateComparator,
+          minWidth: this.currentTheme?.theme?.name === 'modern' ? 180 : 160,
```

Comment:
Should this value also update when the theme changes? The fact that we only set a local variable and don't update the grid when settings change indicate to me a "onceability" issue.